File: src/main/java/Color_yr/AllMusic/API/ISide.java
package Color_yr.AllMusic.API;

import net.md_5.bungee.api.chat.ClickEvent;

public interface ISide {
    void Send(String player, String data, Boolean isplay);

    void Send(String data, Boolean isplay);

    int GetAllPlayer();

    void SendLyric(String data);

    void bq(String data);

    void bqt(String data);

    boolean NeedPlay();

    void SendMessaget(Object obj, String Message);

    void SendMessage(Object obj, String Message);

    void SendMessage(Object obj, String Message, ClickEvent.Action action, String Command);

    void RunTask(Runnable run);

    void reload();

    boolean checkPermission(String player, String permission);
}

File: src/main/java/coloryr/allmusic/api/ISide.kt
package coloryr.allmusic.api

interface ISide {
    fun send(data: String?, player: String?, isplay: Boolean?)
    fun send(data: String?, isplay: Boolean?)
    val allPlayer: Int
    fun bq(data: String?)
    fun bqt(data: String?)
    fun NeedPlay(): Boolean
    fun sendHudLyric(data: String?)
    fun sendHudInfo(data: String?)
    fun sendHudList(data: String?)
    fun sendHudSaveAll()
    fun clearHud(player: String?)
    fun clearHudAll()
    fun sendMessaget(obj: Any?, Message: String?)
    fun sendMessage(obj: Any?, Message: String?)
    fun sendMessageRun(obj: Any?, Message: String?, end: String?, command: String?)
    fun sendMessageSuggest(obj: Any?, Message: String?, end: String?, command: String?)
    fun runTask(run: Runnable?)
    fun reload()
    fun checkPermission(player: String?, permission: String?): Boolean
    fun task(run: Runnable?, delay: Int)
}